cypress/integration/version/action_buttons.ts

```diff
@@ -52,7 +58,7 @@ describe("Action Buttons", () => {
         cy.dataCy("unschedule-patch").click();
         cy.dataCy("abort-checkbox").check({ force: true });
         cy.get(popconfirmYesClassName).contains("Yes").click({ force: true });
-        cy.dataCy("toast").should("exist");
+        cy.dataCy("toast").contains("Success!").should("exist");
```

Comment:
It may be preferable to include the variant type in the toast's data-cy prop since we're often checking for a string to verify the type. No worries if you'd rather not tackle that here though!

Reply:
done!